src/include/uapi/ipc.h

```diff
@@ -301,8 +301,6 @@ struct sof_ipc_dai_ssp_params {
 	uint16_t tdm_per_slot_padding_flag;
 	uint32_t clks_control;
 	uint32_t quirks; // FIXME: is 32 bits enough ?
-	/* private data, e.g. for quirks */
```

Comment:
missing reserved field here, and FIXME above can be removed

Reply:
We don't need a reserved here as there is nor variable length data at the end of the struct, meaning it's safe to add new data to the end (and only update the ABI minor).

Reply:
I see. for all the variable length parts you would however need padding, e.g. sof_ipc_dai_dmic_pdm_ctrl. This is an array so you can't just add fields in there.